### PR TITLE
Localized: Dashboard Monitor Page

### PR DESCRIPTION
**Pull request by ilanus**

### CODE REVIEW

File: src/components/ActiveChart/index.js [hunk 1]
```diff
@@ -53,7 +54,20 @@ export default class ActiveChart extends Component {
...
+            <FormattedMessage
+              id="component.activeChart.target-assessment"
+              defaultMessage="Target assessment"
+            />
+          }
+          total={
+            <FormattedMessage
+              id="component.activeChart.expected-to-reach"
+              defaultMessage="Expected to reach"
+            />
```
Comment (chenshuai2144) [inline src/components/ActiveChart/index.js]: components should not use umi/plugin

Comment (ilanus) [reply src/components/ActiveChart/index.js]: @chenshuai2144 so they cannot be localized? If so I will remove it

**Comment by chenshuai2144** [comment]
rebase master

**Comment by ilanus** [comment]
> rebase master

done

**Pull request merged**